src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = 0;
+
+  return people.reduce((sum, person) => {
+    if (person.sex === 'm'
```

Comment:
Don't use reduce method to filter the data. There is another proper array method for filtering information.